src/phases/execution/MixedMultiDeletes.yml

```diff
@@ -0,0 +1,304 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/server-execution"
+Description: >
+  Runs mass deletions over a set of 1KB documents, then mass deletions over a set of larger, 10MB documents.
+  For each size of document:
+    . Populates a collection with documents of size n-bytes, then performs a mass deletion on the collection.
+    . Repopulates a collection with documents of size n-bytes, then performs a mass deletion while
+      performing concurrent writes on another collection.
+  Allows for comparing the performance of the DELETE_STAGE vs the BATCHED_DELETE_STAGE, primarily in
+  terms of deletion throughput, and w:majority latencies of the concurrent writes. Also, allows for
+  comparisons in performance when mass deletions are performed over different sized documents.
+  The mass deletion namespace is test.Collection0
+  The concurrent writes namespace is test.concurrentWritesColl
+
+Keywords:
+- RunCommand
+- Loader
+- LoggingActor
+- CrudActor
+- insert
+- delete
+- batch
+- deleteMany
+- latency
+
+GlobalDefaults:
+
+  #### User Defined Parameters ####
+
+  # Whether to use the BATCHED_DELETE query exec stage over DELETE.
+  UseBatchedDeleteStage: &UseBatchedDeleteStage {^Parameter: {Name: "UseBatchedDeleteStage", Default: 0}}
+
+  # Configuration file to use: with no secondary indexes (default) or creating secondaries for the non-padding fields.
+  ConfigFile: &ConfigFile {^Parameter: {Name: "ConfigurationFile", Default: "../../phases/execution/config/MultiDeletes/Default.yml"}}
+
+  ##### Constants ####
+
+  WriteRate: &WriteRate 1 per 500 microseconds  # 2000/second
+  ConcurrentWritesColl: &ConcurrentWritesColl concurrentWritesColl
+  Database: &Database test
+  MassDeletionsColl: &MassDeletionsColl Collection0
+  NumPhases: &NumPhases 11
+  LoaderThreads: &LoaderThreads 4
+  # Delete by primary index always.
+  DeleteQuery: &DeleteQuery
+    {_id: {$gte: {^ObjectId: "000000000000000000000000"}}}
+  # Only used for generating the Document padding.
+  RandomStringLength: &RandomStringLength 6
+
+  #### Document Size Specifications ####
+
+  # __________  1KB Specifications __________
```

Comment:
Taken from parameters in [MultiDeletes<>_1KB.yml](https://github.com/mongodb/genny/blob/6d4f0888d97bf5a753460db13e5974cfffaeae17/src/workloads/execution/MultiDeletesBatched_1KB.yml#L25-L30)